core/src/stores/postgres.rs

```diff
@@ -1571,9 +1572,13 @@ impl Store for PostgresStore {
         for row in rows {
             let hash: String = row.get(0);
             let created: i64 = row.get(1);
+            let status_str: String = row.get(2);
+            let status = DocumentStatus::from_str(&status_str)?;
+
             versions.push(DocumentVersion {
                 hash,
                 created: created as u64,
+                status,
```

Comment:
cc @fontanierh document tracker impact: we will return the status when querying the versions of a document. We used to return only hash and created. This endpoint returns deleted version.